relay-general/src/pii/snapshots/relay_general__pii__convert__tests__contexts.snap

```diff
@@ -76,15 +76,7 @@ expression: data
       "the_secret": "hello",
       "type": "runtime"
     },
-    "secret": {
-      "a_password_here": "[Filtered]",
-      "apiKey": "[Filtered]",
-      "api_key": "[Filtered]",
-      "foo": "bar",
-      "password": "[Filtered]",
-      "the_secret": "[Filtered]",
-      "type": "[Filtered]"
-    },
```

Comment:
I have no personal objections against this change, but there seems to have been a product decision at some point in the past to not scrub entire objects in general. There is an entire section in our [docs](https://docs.sentry.io/product/data-management-settings/scrubbing/server-side-scrubbing/) about it. I guess we can do two things:

1. Just go ahead with this change and update the docs.
2. Only add `$object` to specific fields, e.g. `password` and `secret`, but not others like `credentials`, `auth`.

Reply:
If we add `$object` to `password`, why not do the same with `credentials`?

Reply:
The test was quite funny in first place, why the `type` field was filtered here, any ideas?

Reply:
> If we add $object to password, why not do the same with credentials?

There's an example in the docs that lists a `credentials` object which also contains `lastLogin`, a field that does not need to be scrubbed. But I agree that there is no obvious line we can draw between "this should be scrubbed completely" and "this should be scrubbed partially".

Reply:
> why the type field was filtered here, any ideas?

`type` was filtered because it's value was `secret` (compare to other contexts where `type` also corresponds to the top level key).

Reply:
I only now realize that with this change, we will drop entire `Context`s. This goes against the purpose of adding contexts to events (see [docs](https://docs.sentry.io/platforms/javascript/enriching-events/context/)). But I guess users can always rename them if they do not want them scrubbed.